### ISSUE TITLE
Detect when IE/Edge is in High Contrast and automatically switch the theme to hc-black

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/monaco-editor#157]
See Microsoft/vscode#3690, Microsoft/vscode#9726

**Comment by alexandrudima** [comment]
We are no longer supporting IE10.

**Comment by aarms** [comment]
Any ETA?

**Comment by alexandrudima** [comment]
I don't know if in IE there is any JS event available when the OS goes into high contrast mode

**Comment by alexandrudima** [comment]
Fixed with the fix for #472
